### PR TITLE
Add metadata for F-Droid

### PR DESCRIPTION
**Pull request by AMDG2**

### CODE REVIEW

File: fastlane/metadata/android/en-US/short_description.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+A simple and discrete daily reading application (french only)
```
Comment (yadutaf) [inline fastlane/metadata/android/en-US/short_description.txt]: :+1: for the "French only mention"

File: fastlane/metadata/android/en-US/title.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+AELF
```
Comment (yadutaf) [inline fastlane/metadata/android/en-US/title.txt]: Even though the app is not available in English, maybe we can use a name equivalent to French like "AELF - Bible and day's reading" ?

**Comment by AMDG2** [comment]
It is definitely possible. In theory thoses metadata files should first come from what is in the Play-Store. Checkout the [Fastlane Supply documentation](https://docs.fastlane.tools/actions/supply/). This is the format I used to create those metadata.

File: fastlane/metadata/android/en-US/title.txt (hunk 2 above)
Comment (AMDG2) [reply fastlane/metadata/android/en-US/title.txt]: done ;)

**Comment by yadutaf** [comment]
That tool is exactly what we need :)

**Pull request merged**